### PR TITLE
symfony 2.7 requires at least php 5.3.9

### PR DESCRIPTION
**Pull request by scaytrase**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.7 |
| Fixed tickets | none |

https://github.com/symfony/symfony/blob/2.7/composer.json

### CODE REVIEW

**Comment by xabbuh** [comment]
:+1: good catch @scaytrase

File: reference/requirements.rst [hunk 1]
```diff
@@ -29,7 +29,7 @@ Required
 .. caution::
 
     Be aware that Symfony has some known limitations when using a PHP version
-    less than 5.3.8 or equal to 5.3.16. For more information see the
+    equal to 5.3.16. For more information see the
```
Comment (WouterJ) [inline reference/requirements.rst]: This sentence has now become a bit strange. What about: *"Be aware that Symfony has some known limitations when using PHP 5.3.16."* ?

**Comment by scaytrase** [comment]
@WouterJ Done

**Comment by xabbuh** [comment]
@scaytrase Can you please rebase your changes on the latest `2.7` branch?

**Comment by scaytrase** [comment]
@xabbuh Not sure if I done all correctly, check it please.

**Comment by xabbuh** [comment]
@scaytrase Looks good :+1:

**Comment by weaverryan** [comment]
Thanks a lot Pavel!

**Pull request merged**